lib/http_server.js

```diff
@@ -44,22 +44,22 @@ var ZettaHttpServer = module.exports = function(zettaInstance) {
     ssl: false
   });
 
+  var ValidWSUrls = [
+      /^\/events$/, // /events
+      /^\/events?(.+)$/, // /events?topic=query:where type="led"
```

Comment:
Won't this match `/event12345`?  I think you want the following regular expression: `^\/events\?.+$`.